src/Collector/EncountCollector.php

```diff
@@ -41,11 +47,10 @@ public function build($code, $description, $file, $line, $context)
             $description = $exception->getMessage();
             $file = $exception->getFile();
             $line = $exception->getLine();
-            $trace = Debugger::formatTrace($exception, ['format' => 'base']);
+            $trace = Debugger::formatTrace($exception, ['format' => 'text']);
```

Comment:
"base" is missing
https://github.com/cakephp/cakephp/blob/5.x/src/Error/Debugger.php#L418